### PR TITLE
Update controller.rst

### PR DESCRIPTION
**Pull request by ifdattic**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.3 |
| Fixed tickets |  |

### CODE REVIEW

File: book/controller.rst [hunk 1]
```diff
@@ -5,7 +5,7 @@ Controller
 ==========
 
 A controller is a PHP function you create that takes information from the
-HTTP request and constructs and returns an HTTP response (as a Symfony
+HTTP request to construct and return an HTTP response (as a Symfony
```
Comment (WouterJ) [inline book/controller.rst]: I'm not so sure about this change.

Comment (xabbuh) [inline book/controller.rst]: What about "to create and return"?

Comment (WouterJ) [inline book/controller.rst]: Well, it's more about grammar than word usage. I think *A controller* is still the one doing stuff:

A controller:
* takes information from the HTTP request
* constructs an HTTP response
* returns the HTTP response

So imo, it's still "takes information from the HTTP request and creates and returns an HTTP response".

Comment (xabbuh) [inline book/controller.rst]: Indeed, sounds better this way.

**Comment by xabbuh** [comment]
ping @ifdattic :)

**Comment by ifdattic** [comment]
@xabbuh should I revert line 8?

**Comment by xabbuh** [comment]
@ifdattic I think you can replace it with @WouterJ's suggestion: "takes information from the HTTP request and creates and returns an HTTP response" (this replaces "constructs" with "creates").

**Comment by WouterJ** [comment]
rebased this PR in https://github.com/symfony/symfony-docs/pull/5143

**Pull request closed without merging**